src/index.ts

```diff
@@ -41,6 +48,19 @@ export function getRollingMinor(branch: string): string {
   if (match && match.groups) {
     const versionPart = match.groups.version;
     if (versionPart) {
+      // Check that NixOS/nixpkgs has the tag nixos-${versionPart}
+      try {
+        await octokit.rest.git.getRef({
+          owner: "NixOS",
+          repo: "nixpkgs",
+          ref: versionPart,
```

Comment:
```suggestion
          ref: `tags/versionPart`,
```

...but note: this won't exist for `unstable` releases.